api/config/config_local.json

```diff
@@ -2,19 +2,24 @@
 {
   "firecloud": {
     "baseUrl": "https:\/\/firecloud-orchestration.dsde-dev.broadinstitute.org",
-    "billingAccountId": "014D91-FCB792-33D2C0",
-    "billingProjectPrefix": "aou-rw-local1-",
     "clusterMaxAgeDays": 7,
     "clusterIdleMaxAgeDays": 3,
     "debugEndpoints": false,
     "enforceRegistered": true,
     "leoBaseUrl": "https:\/\/leonardo.dsde-dev.broadinstitute.org",
     "registeredDomainName": "all-of-us-registered-test",
-    "billingRetryCount": 2,
-    "billingProjectBufferCapacity": 2,
     "xAppIdValue": "local-AoU-RW",
     "vpcServicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test"
   },
+  "billing": {
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-local1-",
+    "billingExportBigQueryTable": "all-of-us-workbench-test-bd.billing_data.gcp_billing_export_v1_014D91_FCB792_33D2C0",
+    "billingRetryCount": 2,
+    "billingProjectBufferCapacity": 2,
+    "defaultFreeCreditsLimit": 300.0,
+    "whitelistedUsers": []
```

Comment:
Whitelisted for what exactly? I.e. you might want multiple whitelists for different billing-related things at some point.

Also, does keeping this here imply we need to redeploy the config to add users to the whitelist, or can that be done while we're running?